Makefile

```diff
@@ -38,7 +38,7 @@ install:
 	@python3 -m venv venv
 	@source $(CURDIR)/venv/bin/activate && \
 		pip install --upgrade pip && \
-	    pip install -e .[lint,test,dev,docs,plots] && \
+	    pip install --no-cache-dir -e .[lint,test,dev,docs,plots] && \
 	    deactivate
```

Comment:
this forces us to not use the cache. this is useful to avoid errors when hot swapping low-level modules. this only affects me, users never see this